backend/src/main/java/com/gaucho/sos/repositories/UserRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.gaucho.sos.repositories;
+
+import com.gaucho.sos.domain.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
```

Comment:
Não daria pra remover essa linha em branco? Só pra manter todos os `import` juntos.
```suggestion
import java.util.Optional;
```

Reply:
Alinhado que vamos manter dessa forma.